CONTRIBUTING.md

```diff
@@ -0,0 +1,173 @@
+# Contributing to UINotifications
+
+As the creators, and maintainers of this project, we're glad to share our projects and invite contributors to help us stay up to date. Please take a moment to review this document in order to make the contribution process easy and effective for everyone involved.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue or assessing patches and features.
+
+In general, we expect you to follow our [Code of Conduct](https://github.com/WeTransfer/UINotifications/blob/master/CODE_OF_CONDUCT.md).
+
+## Using the issue tracker for bug reports, feature requests and discussions
+
+### First time contributors
+We should encourage first time contributors. A good inspiration on this can be found [here](http://www.firsttimersonly.com/). As pointed out:
+
+> If you are an OSS project owner, then consider marking a few open issues with the label first-timers-only. The first-timers-only label explicitly announces:
+
+> "I'm willing to hold your hand so you can make your first PR. This issue is rather a bit easier than normal. And anyone who’s already contributed to open source isn’t allowed to touch this one!"
+
+By labeling issues with this `first-timers-only` label we help first time contributors step up their game and start contributing.
+
+### Bug reports
+
+A bug is a _demonstrable problem_ that is caused by the code in the repository.
+Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports:
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been
+   reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
+   latest `master` or development branch in the repository.
+
+3. **Isolate the problem** &mdash; provide clear steps to reproduce.
+
+A good bug report shouldn't leave others needing to chase you up for more
+information. Please try to be as detailed as possible in your report. What is
+your environment? What steps will reproduce the issue? What would you expect to be the outcome? All these details will help people to fix any potential bugs.
+
+Example:
+
+> Short and descriptive example bug report title
+>
+> A summary of the issue and the OS environment in which it occurs. If
+> suitable, include the steps required to reproduce the bug.
+>
+> 1. This is the first step
+> 2. This is the second step
+> 3. Further steps, etc.
+>
+> `<url>` - a link to the reduced test case, if possible
+>
+> Any other information you want to share that is relevant to the issue being
+> reported. This might include the lines of code that you have identified as
+> causing the bug, and potential solutions (and your opinions on their
+> merits).
+
+### Feature requests
+
+Feature requests are welcome. But take a moment to find out whether your idea
```

Comment:
Just like for bugs, I think that it might be worth mentioning here that contributors need to check if the feature request already exist. If it does, I think that we should recommend using the thumbs up reaction emote, as a way to show that there is more interest. 
Fighting duplicates is something I've seen a lot, even for feature request.

Reply:
Yup, good point.